.cargo/config

```diff
@@ -1,3 +1,7 @@
+#dont know if I want a build section or a ENV yet....
+#[build]
+#rustc-wrapper = "sccache"
+
```

Comment:
We probably would want this in the x config not in cargo's, that way we aren't breaking folks running bare cargo commands.